SpatialGDK/Source/SpatialGDK/Public/Interop/SpatialStaticComponentView.h

```diff
@@ -47,7 +47,12 @@ class SPATIALGDK_API USpatialStaticComponentView : public UObject
 
 	void GetEntityIds(TArray<Worker_EntityId_Key>& OutEntityIds) const { EntityComponentMap.GetKeys(OutEntityIds); }
 
+	void SetOverridenInterestComponent(Worker_EntityId_Key, uint32_t);
+	int32_t GetOverridenInterestComponent(Worker_EntityId_Key);
+	void ClearOverridenInterestComponent(Worker_EntityId_Key);
+
 private:
+	TMap<Worker_EntityId_Key, uint32_t> OverridenInterestComponent;
```

Comment:
This process is nice and elegant, but I think falls down with actor transitioning between servers. Seems it'd be nicer if we could just check the static view for which component current lives on the entity?